eslint.config.mjs

```diff
@@ -0,0 +1,52 @@
+import globals from 'globals';
+import pluginJs from '@eslint/js';
+import prettierPlugin from 'eslint-plugin-prettier';
+
+const { node, browser } = globals;
+
+export default [
+  pluginJs.configs.recommended,
+  {
+    files: ['**/*.js'],
+    plugins: {
+      prettier: prettierPlugin,
+    },
+    languageOptions: {
+      sourceType: 'commonjs',
+      globals: {
+        ...node,
+        ...browser,
```

Comment:
One doubt here, I think Discord is only server side. Why is browser needed here?